src/TypeStringifier.php

```diff
@@ -430,6 +430,10 @@ private function stringifyShapeElement(bool $list, int|string $key, ShapeElement
             return $element->type->accept($this);
         }
 
+        if (\is_string($key) && ($key === '' || preg_match('/\W/', $key))) {
+            $key = str_replace("\n", '\n', var_export($key, true));
```

Comment:
Sorry, I've changed my mind! Let's actually make `escapeStringLiteral` method and use it for `visitStringLiteral` as well! +add a test that `types::stringliteral("\n")` looks nice.

Reply:
You will have only  `return str_replace("\n", '\n', var_export($key, true));` in that method.